src/app/components/Disclaimer/index.test.tsx

```diff
@@ -43,60 +39,51 @@ const renderComponent = (
         },
       }}
     >
+      {/* @ts-expect-error - to diagnose */}
       <ServiceContext.Provider value={{ disclaimer, externalLinkText }}>
         <DisclaimerComponent />
       </ServiceContext.Provider>
     </ToggleContextProvider>,
   );
-
 describe('Disclaimer Component', () => {
   it('should render a section with role region', () => {
-    const { getByRole } = renderComponent();
-    expect(getByRole('region', { container: 'section' })).toBeInTheDocument();
+    const { container } = renderComponent();
+    const region = screen.getByRole('region');
+    expect(region).toBeInTheDocument();
+    const section = container.querySelector('section');
+    expect(section).toBeInTheDocument();
   });
-
   it('should render disclaimer text correctly', () => {
     const { getByText } = renderComponent();
     expect(
       getByText(
         'Приложение Русской службы BBC News доступно для и . Вы можете также подписаться на наш канал в .',
       ),
     ).toBeInTheDocument();
-
     expect(getByText('IOS')).toBeInTheDocument();
-
     expect(getByText('Android')).toBeInTheDocument();
-
     expect(getByText('Telegram')).toBeInTheDocument();
   });
-
   it('should render links correctly', () => {
     const { getAllByRole } = renderComponent();
     expect(getAllByRole('link').length).toBe(3);
   });
-
   it('should not render the disclaimer when the disclaimer toggle is not enabled', () => {
     const { container } = renderComponent({ enabled: false });
     expect(container).toBeEmptyDOMElement();
   });
-
   it('should not render when disclaimer is null', () => {
+    // @ts-expect-error - not expecting null
     const { container } = renderComponent({ enabled: true }, null);
     expect(container).toBeEmptyDOMElement();
   });
-
   it('should not render when disclaimer is empty object', () => {
+    // @ts-expect-error - not expecting empty object
     const { container } = renderComponent({ enabled: true }, {});
     expect(container).toBeEmptyDOMElement();
   });
-
   it('should render links with external label if links are external', () => {
     const { getByLabelText } = renderComponent();
     expect(getByLabelText('IOS, внешняя')).toBeInTheDocument();
   });
-
-  it('should not render links with external label if isExternal is false', () => {
```

Comment:
Did we mean to delete this test?

Reply:
I'm using the newer inlineLink component which doesn't seem to take the isExternal prop. (If I understand correctly, it's worked out here: https://github.com/bbc/simorgh/blob/latest/src/app/components/InlineLink/index.tsx#L39

So I didn't think the test was relevant.